src/ecies/mod.rs

```diff
@@ -91,8 +91,21 @@ use crate::Curve25519PublicKey;
 
 mod messages;
 
-const ENCRYPTION_KEY_S_INFO: &str = "MATRIX_QR_CODE_LOGIN_ENCKEY_S";
-const ENCRYPTION_KEY_G_INFO: &str = "MATRIX_QR_CODE_LOGIN_ENCKEY_G";
+const MATRIX_INFO_PREFIX: &str = "MATRIX_QR_CODE_LOGIN";
```

Comment:
```suggestion
const MATRIX_QR_LOGIN_INFO_PREFIX: &str = "MATRIX_QR_CODE_LOGIN";
```

Maybe make it more explicit?

Reply:
https://github.com/matrix-org/vodozemac/pull/151/commits/4bdc765c57c77a0d6dd65d78e72bb8e3d4b5fd71